examples/ckks/advanced/rlwe_lwe_bridge_CDKS20/main.go

```diff
@@ -0,0 +1,300 @@
+package main
+
+import (
+	"fmt"
+	"github.com/ldsec/lattigo/v2/ring"
+	"github.com/ldsec/lattigo/v2/rlwe"
+	"github.com/ldsec/lattigo/v2/utils"
+	"math/bits"
+	"time"
+)
+
+// This example is an implementation of the work "Efficient Homomorphic Conversion Between (Ring) LWE Ciphertexts"
+// from Hao Chen and Wei Dai and Miran Kim and Yongsoo Song.
```

Comment:
Need full bibliographic reference as well as a high-level description of what the program is doing.

Reply:
Done